llvm/test/tools/llvm-objcopy/ELF/dump-section-directory-not-exists.test

```diff
@@ -0,0 +1,37 @@
+## Show that llvm-objcopy report that section file path is not exists.
```

Comment:
Place the test into the primary test `dump-section.test`. We often use one test file for both positive and negative tests. Sometimes this actually improves test discoverability

Reply:
All comments addressed, thank you